webdriver-ts/src/webdriverAccess.ts

```diff
@@ -193,17 +260,26 @@ export async function clickElementById(driver: WebDriver, id: string, isInButton
   });
 }
 
-export async function clickElementByXPath(driver: WebDriver, xpath: string, isInButtonArea: boolean) {
+export async function clickElementByXPath(
+  driver: WebDriver,
+  xpath: string,
+  isInButtonArea: boolean
+) {
   return await retry(5, driver, async function (driver, count) {
-    if (count > 1 && config.LOG_DETAILS) console.log("clickElementByXPath ", xpath, " attempt #", count);
+    if (count > 1 && config.LOG_DETAILS)
+      console.log("clickElementByXPath ", xpath, " attempt #", count);
```

Comment:
I (hope I) wouldn't use multi line if blocks without braces.

Reply:
It only happened because of the length of the string. If it is less than 100, the function after `if` is not moved to another line